applications/nrf5340_audio/src/modules/dfu_entry.c

```diff
@@ -111,9 +99,14 @@ void dfu_entry_check(ble_init_func ble_init)
 
 	if (pressed) {
 		LOG_INF("Enter SMP_SVR service only status");
-		ret = ble_init(on_ble_core_ready_dfu_entry);
+		ret = ble_init(false);
```

Comment:
If we are using DFU mode, ble_init() will be bt_mgmt_init(), which without input parameter when defined in bt_mgmt.c.
Please make a check.